onnxruntime/python/tools/tensorrt/perf/mem_test/main.cpp

```diff
@@ -3,12 +3,14 @@
 //
 
 #include <assert.h>
+#include <iostream>
 #include <vector>
 #include <onnxruntime_cxx_api.h>
 #include <tensorrt_provider_factory.h>
+#include <tensorrt_provider_options.h>
 
-std::unique_ptr<OrtTensorRTProviderOptions> get_default_trt_provider_options() {
-  auto tensorrt_options = std::make_unique<OrtTensorRTProviderOptions>();
+std::unique_ptr<OrtTensorRTProviderOptionsV2> get_default_trt_provider_options() {
```

Comment:
we aren't supposed to modify the v2 options struct directly. 
we don't need this get_default_trt_provider_options() function anymore after moving to v2 struct. 
you removed the call in line 51.

Reply:
Thanks for pointing that out. This func has been removed as well.